htsget.md

```diff
@@ -206,6 +207,20 @@ The server SHOULD respond with an `InvalidInput` error if any other htsget query
 </td></tr>
 <tr markdown="block"><td>
 
+`encryptionScheme`
+_optional_
+</td><td>
+
+Request data encrypted with a particular encryption scheme.
+
+
+
+If `encryptionScheme` is specified a 4 letter code for the encryption standard MUST be returned. When `c4gh` is specified, Crypt4GH payload for a particular object will be returned (if available).
```

Comment:
Do you mean a 4-character code must be specified as `encryptionScheme`'s **value**? I don't understand why such a code would be being **returned** by the server.

Reply:
Agreed, fixed in https://github.com/samtools/hts-specs/pull/808/commits/016f5957d4626ee4ef99916c98e2d62141c3149e